assets/js/geoContrastCountries/Country.js

```diff
@@ -95,6 +94,316 @@ function openCountryPopup (feat) {
     };
 };
 
+    }
```

Comment:
@calvinb4 I fixed an orphaned end-bracked that was introduced here, and also commented out the other code below, since it also introduced errors. See cb1783fa4cc387a61afa76ed1760347c673e975e. 

Either way, this Country.js file is in the geoContrastCountries which is reserved for "Global Boundaries" tab. I suspect you were actually intending to make changes for the new "Data" tab instead? If so, then you should move the code I commented out to the "geoData" folder which I see you've created.

Reply:
Thanks, I think the pull request I just opened (https://github.com/wmgeolab/gbWeb/pull/13#issue-1159897390) should resolve the issues while adding the Data tab back to the website.